src/parser/deathknight/unholy/modules/features/ScourgeStrikeEfficiency.tsx

```diff
@@ -50,7 +59,7 @@ class FesteringStrikeEfficiency extends Analyzer {
     }
   }
 
-  suggestions(when) {
+  suggestions(when: When) {
     const percentCastZeroWounds = this.zeroWoundCasts/this.totalCasts;
     const strikeEfficiency = 1 - percentCastZeroWounds;
     when(strikeEfficiency).isLessThan(0.80)
```

Comment:
Same as before with removing magic numbers and creating a get suggestionThreshold type function to retrieve that information